lesson_3_hw/lesson_3_hw.py

```diff
@@ -0,0 +1,176 @@
+"""- Implement to boolean function. Retrieve different type of object returns
+True or False"""
+
+
+def is_empty(cart_items_added, cart_items):
+    """
+The function checks if the cart is empty.
+    :param cart_items_added: list of recently added items
+    :param cart_items: list of existed cart items
+    :return: bool
+    """
+    if len(cart_items_added) == 0 and len(cart_items) == 0:
+        result = True
+    else:
+        result = False
+
+    return result
+
+
+print(is_empty(cart_items_added=["headphones"],
+               cart_items=[]))
+
+
+def is_student(student):
+    """
+The function checks if the person is a listed student.
+    :param student: name of the person
+    :return: bool
+    """
+    students = ["Steve", "Matt", "Ellie", "Dan", "Mark", "Angela"]
+    if student in students:
+        result = True
+    else:
+        result = False
+
+    return result
```

Comment:
You can use "return student in students"

Reply:
Fixed